### ISSUE TITLE
Disk-based data

### ISSUE THREAD

**Issue by willtownes** [ChristophH/sctransform#63]
It would be awesome to be able to run sctransform on data that are too big to fit in memory (eg, HDF5 files accessed through something like [DelayedArray](https://bioconductor.org/packages/release/bioc/html/HDF5Array.html)). Is this a feature that would be considered for the package in the future? We are hoping to wrap sctransform to facilitate interoperability with SingleCellExperiment objects via the scry package (kstreet13/scry#14 )

**Comment by ChristophH** [comment]
I agree that it would be awesome, but so far this feature has only been tested in a previous development version of sctransform (through [loomR](https://github.com/mojaveazure/loomR)).
I will look into the DelayedArray and HDF5Array packages to figure out what I need to do to make sctransform work with on-disk data.

**Comment by PedroMilanezAlmeida** [comment]
I believe glmGamPoi can handle DelayedArray out-of-the box, right?

Would it be of interest to have at least the "v2 flavor" workflow compatible with DelayedArray?

**Comment by PedroMilanezAlmeida** [comment]
Just checking if there were any updates on this that I may have missed. Any help would be appreciated.
